### ISSUE TITLE
Overkill doesn't gib zombie corpse

### ISSUE THREAD

**Issue by bakaemon** [CleverRaven/Cataclysm-DDA#60140]
### Describe the bug

It happened in the recent experimental version and the lastest. I found out that almost all the humanoid enemies corpses didn't become pulped after I dealt damage that is twice their base HP.

### Steps to reproduce

1. Debug my character to have really high strength that deal huge damage (250~300) using high bashing damage weapon Heavy sledge hammer on common zombies.
2. The second time I created a reasonable stats character with 14 str only and leave the rest 8. I started with High Tech Low Life scenario and installed myself hydraulic muscle bionic and equiped myself a heavy sledge hammer. Still not work though, again, non-humanoid enemies don't have this issue.

### Expected behavior

Some humanoid enemies, or at least normal zombies, should be pulped and explode into gory messes after receiving overkill damage.

### Versions and configuration

- OS: Android 9
- Version: Experiment bb34fdb
- Mods: None.

**Comment by PatrikLundell** [comment]
I believe I've seen a single case of gibbing (a zombie child) with my 14 STR character.

**Comment by bakaemon** [comment]
> I believe I've seen a single case of gibbing (a zombie child) with my 14 STR character.

Indeed, the zombie child sometimes work fine(but rarely), but that's the only exception.
